test/knot-data/zones/test.zone

```diff
@@ -0,0 +1,19 @@
+; SPDX-License-Identifier: LGPL-2.1-or-later
+$TTL 86400
+$ORIGIN test.
+
+@ IN SOA ns1.signed.test. root.signed.test. (
+    42   ; serial
+    3H   ; refresh
+    15M  ; retry
+    1W   ; expire
+    1D   ; minimum TTL
+)
+
+; NS info
+@                     NS ns1.signed.test.
+ns1.signed            A  1.2.3.1
```

Comment:
For example {a,b,c,d,e,f}.root-servers.net are not signed servers. Perhaps it should copy its behaviour and move common nameserver to unsigned zone. Probably just nice to have, without changing anything.